save-deploy/docker-mac-settings.sh

```diff
@@ -0,0 +1 @@
+socat TCP-LISTEN:2375,range=127.0.0.1/32,reuseaddr,fork UNIX-CLIENT:/var/run/docker.sock
```

Comment:
Why can't orchestrator use UNIX socket directly? Also, can HTTP endpoint be enabled on docker daemon directly, without proxying script? On Linux we can add `-H 0.0.0.0:2375` to flags of `dockerd` command

Reply:
The point it that @icemachined suggested to use docker with `TCP` instead of `UNIX socket`. Talking about `-H` flag it seems that it should work, but there is no `dockerd` command on Mac, probably you meant `docker`

Reply:
> probably you meant `docker`

I honestly know nothing about specifics of Mac. On Linux where Docker is started as a Docker daemon `dockerd` there is a flag; on Windows there is a toggle in UI of Docker Desktop. If Docker for Mac works only with UI too, maybe there is a toggle as well?

Reply:
As discussed, I can see no other workaround